language/types_values_variables.md

```diff
@@ -646,6 +978,28 @@ as negative value.
 
 Defaults to `p`
 
+##### Timespan to String
+
+| Format  | Timespan formats
+| ----    | ------------------
+| s       | formats according to the timespan format string '%D-%H:%M:%S' 
+| p       | programmatic representation - a quoted string formatted according to the timespan format string '%D-%H:%M:%S'
```

Comment:
Hm, that is not an accurate programmatic representation - it would need to say Timestamp('...'), or Timespan('...') for that to be true since you cannot parse the value back again to the same data type.